apps/hash-graph/lib/graph/src/api/rest/json_schemas/entity_type.json

```diff
@@ -10,63 +10,150 @@
       ]
     },
     "kind": {
+      "type": "string",
       "enum": ["entityType"]
     },
     "$id": {
-      "type": "string",
-      "format": "uri"
+      "$ref": "#/definitions/VersionedUrl"
     },
     "type": {
+      "type": "string",
       "enum": ["object"]
     },
-    "title": { "type": "string" },
-    "description": { "type": "string" },
+    "title": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string"
+    },
+    "allOf": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/EntityTypeReference"
+      }
+    },
     "examples": {
-      "$comment": "Example Entity instances",
```

Comment:
Why removing the comments?

Reply:
and because of that I decided to remove them (as there are no tools that recognize `x-$comment`), when we switch to OpenAPI 3.1, we could simply embed the schema.